node/coinstacks/gnosis/daemon/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+start_coin_bg() {
```

Comment:
i've been using `start()` and `stop()` on more recent init scripts, doesn't matter too much, but just to make note